.github/workflows/default.yml

```diff
@@ -131,7 +132,7 @@ jobs:
           MYSQL_PASSWORD: links
           MYSQL_ROOT_PASSWORD: links
           MYSQL_DATABASE: links
-          MYSQL_ROOT_HOST: '%'
+          MYSQL_ROOT_HOST: "%"
```

Comment:
I don't understand the necessity of this change.

Reply:
It must have been my linter